### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Rpgdudester** [mysql-net/MySqlConnector#1438] [General]
Hey @bgrainger, I want to thank you and whoever has worked with you on this library!

It's been great to be able to drop something in and just have it work. I've been using it on a SaaS c# middleman for my boss and besides a few minor issues (mostly async/await stuff on my side because I'm still learning), it's been great.

So I just wanted to say thanks!

-Joshua

**Comment by bgrainger** [comment]
Thanks for the feedback!
